tests/helpers.go

```diff
@@ -191,7 +191,7 @@ func executeTest(t *testing.T, testFile string) {
 			var exitError *exec.ExitError
 			if errors.As(err, &exitError) {
 				if exitError.ExitCode() == 1 {
-					require.Fail(t, string(out))
+					require.Fail(t, err.Error())
```

Comment:
can we print both, because I believe the output will have information from the execution

Reply:
```go
out, err := cmd.CombinedOutput()
t.Log(string(out))
```
The output is already printed, outside the condition.